src/style.css

```diff
@@ -1 +1,44 @@
-/* add styles here */
+.page {
+  margin: 0;
+  padding: 0;
+}
+
+.page__body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('../images/star.svg');
+  background-size: cover;
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
True- remove it:
![image](https://github.com/user-attachments/assets/0ae7097e-4b8c-4c94-b51d-4bc2f09b5361)